### PR TITLE
docs(databases): Add permissions how-to page int-add-databases

### PR DESCRIPTION
**Pull request by bastienwirtz**

### CODE REVIEW

File: storage/database/how-to/manage-permissions.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+meta:
+  title: How to manage user permissions for Managed Databases
+  description: This page explains how to manage user permissions on a Managed Database
+content:
+  h1: How to manage user permissions for Managed Databases
+  paragraph: This page explains how to manage user permissions on a Managed Database
+dates:
+  validation: 2022-03-30
```
Comment (bene2k1) [inline storage/database/how-to/manage-permissions.mdx:9]:
```suggestion
  validation: 2022-03-25
```

**Review by bene2k1 — APPROVED** [review]

File: storage/database/how-to/manage-permissions.mdx (hunk 1 above)
Comment (bastienwirtz) [reply storage/database/how-to/manage-permissions.mdx]: Fixed!

**Comment by abarbare** [comment]
👍

**Review by ofranc — APPROVED** [review]

File: storage/database/how-to/manage-permissions.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  description: This page explains how to manage user permissions on a Managed Database
+content:
+  h1: How to manage user permissions for Managed Databases
+  paragraph: This page explains how to manage user permissions on a Managed Database
+dates:
+  validation: 2022-03-25
+  posted: 2022-03-25
+---
+
+Permissions define the actions a user can perform on a logical database. By default, no permission are given to a newly created user, except for the initial user created during the instance configuration (which is granted all permissions).
```
Comment (ldecarvalho-doc) [inline storage/database/how-to/manage-permissions.mdx:13]:
```suggestion
Permissions define the actions a user can perform on a logical database. A database user is generated automatically when you create your Database Instance and this user is attributed all permissions. However, newly created users have no permissions by default. 
```

File: storage/database/how-to/manage-permissions.mdx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+  <Message type="note">
+
+  Three types of permissions are available:
+    - **None:** No access to the database
+    - **Read:** Allow users to read tables and fields in a database
+    - **Write:** Allow users to write content in databases
+
+  </Message>
+7. Click **Update user** to confirm.
```
Comment (ldecarvalho-doc) [inline storage/database/how-to/manage-permissions.mdx:39]:
```suggestion
6. Click **Update user** to confirm. 
```

File: storage/database/how-to/manage-permissions.mdx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+7. Click **Update user** to confirm. 
+
+## Set permissions for a new user
+
+During user creation, the permissions panel can be accessed through the top tab menu. 
+For more information see the [add users how to](/database/how-to/add-users/)
+
+<Message type="note">
+
+Permissions are applied on existing objects **at the moment the configuration is set**, and not applied to any object created afterward.
```
Comment (ldecarvalho-doc) [inline storage/database/how-to/manage-permissions.mdx:48]:
```suggestion
Permissions are applied on existing items **at the moment the configuration is set**, and not applied to any items created afterward. 
```

File: storage/database/how-to/manage-permissions.mdx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+## Set permissions for a new user
+
+During user creation, the permissions panel can be accessed through the top tab menu. 
+For more information see the [add users how to](/database/how-to/add-users/)
+
+<Message type="note">
+
+Permissions are applied on existing objects **at the moment the configuration is set**, and not applied to any object created afterward. 
+Permissions are automatically set to "**Custom**" when the database structure was modified after the permission was set. "**Custom**" means the initial permission is partially applied.
```
Comment (ldecarvalho-doc) [inline storage/database/how-to/manage-permissions.mdx:49]: **Custom** means the initial permissions are partially applied. -- What does it mean to have the permissions partially applied? What happens to the other part?
```suggestion
When you modify your database configuration, your configured permissions are automatically set to **Custom**. **Custom** means the initial permissions are partially applied.
```

Comment (bastienwirtz) [reply storage/database/how-to/manage-permissions.mdx:49]: :wave: 
It means the user does not have the permission applied to the new items. The user will still have the initial permission on the old items and But the `None` permission on the new ones (like if no permission was set).

Comment (ldecarvalho-doc) [reply storage/database/how-to/manage-permissions.mdx:49]: Ok, got it. Thank you! I added to my suggestion accordingly 🙏🏻

**Comment by bastienwirtz** [comment]
Thanks for the feedback @ldecarvalho-doc, it's all fixed!

**Review by ofranc — APPROVED** [review]

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**